scripts/json2synapse.py

```diff
@@ -0,0 +1,217 @@
+import os
+import sys
+import pandas
+import numpy
+import synapseclient
+from synapseclient import Table
+from pandas.io.json import json_normalize
+import logging
+
+syn = synapseclient.login()
+
+# TODO: replace with auto download of raw files in repo
```

Comment:
Use this url to find json files in the repo (returns as json):

https://api.github.com/repositories/61157778/contents/synapseAnnotations/data?ref=master